cmd/meroxa/root/environments/repair.go

```diff
@@ -81,7 +83,24 @@ func (r *Repair) Execute(ctx context.Context) error {
 		return err
 	}
 
-	r.logger.Infof(ctx, `The repairment of your environment %q is now in progress and your environment will be up and running soon.`, r.args.NameOrUUID) // nolint:lll
+	state := rr.Status.State
+	name := rr.Name
+	if state == meroxa.EnvironmentStatePreflightError {
+		text := fmt.Sprintf("Environment %q could not be repaired because it failed the preflight checks.", name)
+		details := utils.EnvironmentPreflightTable(rr)
+		text += fmt.Sprintf("\n%s\n", details)
+
+		r.logger.Errorf(ctx, text)
```

Comment:
I recommend staying consistent between all operations. This is an operation code taken from `create` that seems a little neater
```suggestion

               details := utils.EnvironmentPreflightTable(rr)
               c.logger.Errorf(ctx,
			"Environment %q could not be repaired because it failed the preflight checks\n%s\n",
			environment.Name,
			details)
 
```